pkgs/development/libraries/wayland/scanner.nix

```diff
@@ -0,0 +1,48 @@
+{
+  lib,
+  stdenv,
+  wayland,
+  meson,
+  pkg-config,
+  ninja,
+  wayland-scanner,
+  expat,
+  libxml2,
+}:
+
+stdenv.mkDerivation {
+  pname = "wayland-scanner";
+  inherit (wayland) version src;
+
+  outputs = [
+    "out"
+    "bin"
+    "dev"
+  ];
+  separateDebugInfo = true;
+
+  mesonFlags = [
+    "-Ddocumentation=false"
+    "-Dlibraries=false"
+    "-Dtests=false"
```

Comment:
nit: `lib.mesonBool`

Reply:
Sure, applied here and in `default.nix, no rebuild.